fe/fe-core/src/main/java/com/starrocks/persist/OperationType.java

```diff
@@ -88,6 +88,9 @@ public class OperationType {
     @IgnorableOnReplayFailed
     public static final short OP_DROP_PARTITION = 13;
 
+    @IgnorableOnReplayFailed
```

Comment:
Newly added operationType should be greater than 10000, put it after `OP_RECOVER_PARTITION_VERSION` with value 10246.